test/EmptyApp20.FunctionalTests/FunctionalTest/RequestTelemetryEmptyAppTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace EmptyApp20.FunctionalTests.FunctionalTest
+{
+    using System.Linq;
+    using System.Net.Http;
+    using FunctionalTestUtils;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Xunit;
```

Comment:
It looks like Sergey was removing XUnit with [this PR](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/608/files), so we might want to use MSTEST here instead as well.

Reply:
The issue faced in other repo is not applicable in aspnet core repro as we don't target anything below net451. 
We can replace it in future if need arises, but as of now i dont see strong reason to remove it. It'll require some work as the output logging is based on xunit now.